### ISSUE TITLE
Restart service after Task 9: Upgrade the data

### ISSUE THREAD

**Issue by NKarolak** [MicrosoftDocs/dynamics365smb-devitpro-pb#967]
Please add the following information:
After upgrading the data (Task 9), a service restart is required, else the Web Client displays an error. 
Task 10 is currently saying "At this point, the upgrade is complete, and you should be able to open the client." which is not true.

Edit: Sorry, I did not see that you have updated the article in the meantime, inserting a new task 7: Restart the Service.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5e435e27-0b74-7dcd-555d-977517fd3774
* Version Independent ID: ea922161-12f9-3f86-69f9-2044369d6574
* Content: [Upgrade an unmodified application - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/upgrade/upgrade-unmodified-application#task-9-upgrade-the-data)
* Content Source: [dev-itpro/upgrade/upgrade-unmodified-application.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/live/dev-itpro/upgrade/upgrade-unmodified-application.md)
* Service: **dynamics365-business-central**
* GitHub Login: @jswymer
* Microsoft Alias: **jswymer**
